adoc/admin-custom-config.adoc

```diff
@@ -0,0 +1,94 @@
+[#addon-kustomize]
+= Custom Addon Configuration
+
+[NOTE]
+====
+If you use multiple management workstations to manage the {productname} cluster, make sure
+that your customizations to the config files are synced to all management workstations.
+
+Otherwise, if the set of customizations is outdated, the next run of `skuba addon upgrade` will
+undo changes and possibly leave the cluster or some workloads in non-functional condition.
+====
+
+== Using Kustomize
+
+{productname} uses link:https://github.com/kubernetes-sigs/kustomize[Kustomize]
+to allow customization of included addons.
+
+Kustomize allows you to create YAML or JSON files without a template that will override
+the default configurations, leaving the default files intact. For information about
+the file format and allowed fields, refer to: https://kubernetes-sigs.github.io/kustomize/api-reference/glossary/#kustomization
+
+The configuration options specified in the `kustomize` files will be "appended" to the
+default configuration. This, in effect, means that the default options will be effectively overwritten.
+In reality, the options will be parsed from both the default and custom configuration and only the latter applies.
+This can lead to some outputs for commands showing option/parameter flags that are overruled by the custom configuration.
+
+[TIP]
+====
+Using YAML files has the advantage that these files can be directly applied to the cluster
+without converting them first. Using JSON format requires converting the files to YAML with `kustomize build`.
+====
+
+The configuration files are managed in a central location on the management workstation
+together with your cluster definition files that were created during the deployment of the cluster.
+
+----
+.
+├── addons
```

Comment:
This seems to be not in sync with for 4.5 release.. I have this in "addons":

.```
/cri
./cri/conf.d
./cri/conf.d/01-caasp.conf
./cri/conf.d/README
./kucero
./kucero/base
./kucero/base/kucero.yaml
./kucero/patches
./kured
./kured/base
./kured/base/kured.yaml
./kured/patches
./metrics-server
./metrics-server/base
./metrics-server/base/metrics-server.yaml
./metrics-server/patches
./psp
./psp/base
./psp/base/psp.yaml
./psp/patches
./cilium
./cilium/base
./cilium/base/cilium.yaml
./cilium/patches
./dex
./dex/base
./dex/base/dex.yaml
./dex/patches
./gangway
./gangway/base
./gangway/base/gangway.yaml
./gangway/patches
```

Reply:
I removed the tree because we currently don't have a good process for keeping this list updated.